pentaho-gwt-widgets/src/org/pentaho/gwt/widgets/client/colorpicker/TransparencyImpl.java

```diff
@@ -416,7 +416,7 @@ public static void setTransparency( Element elem, int alpha ) {
       DOM.setStyleAttribute( elem, "filter", "alpha(opacity=" + alpha + ")" ); //$NON-NLS-1$ //$NON-NLS-2$ //$NON-NLS-3$
     } else {
       // Everyone else
-      DOM.setStyleAttribute( elem, "-moz-opacity", "" + ( new Integer( alpha ).floatValue() / 100 ) + "" ); //$NON-NLS-1$ //$NON-NLS-2$ //$NON-NLS-3$
+      DOM.setStyleAttribute( elem, "MozOpacity", "" + ( new Integer( alpha ).floatValue() / 100 ) + "" ); //$NON-NLS-1$ //$NON-NLS-2$ //$NON-NLS-3$
```

Comment:
@krivera-pentaho from CSS side you are right, but in GWT we need to use camel case. Look at `com.google.gwt.dom.client.Style#setProperty(java.lang.String, java.lang.String)` - there is `com.google.gwt.dom.client.Style#assertCamelCase` in the first line. According to [Mozilla doc](https://developer.mozilla.org/en-US/docs/Web/CSS/opacity) `-moz-opacity` is an alias for `opacity`. I'm going to use this style attribute if you are OK with it.